src/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
+import * as React from 'react';
 import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
 import MemoIc_arrow_down_vote_off from '../../assets/arrow/Ic_downvote_off';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of imports has been changed. This doesn't affect the functionality, but it's a good practice to keep the import statements organized.
```diff
- import * as React from 'react';
- import PropTypes from 'prop-types';
+ import PropTypes from 'prop-types';
+ import * as React from 'react';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->